plugin/src/main/java/org/jenkinsci/plugins/workflow/cps/CpsStepContext.java

```diff
@@ -435,7 +435,9 @@ public void onFailure(Throwable t) {
                 }
             });
         } catch (IOException x) {
-            LOGGER.log(Level.FINE, null, x);
+            // TODO: If the problem is with the FlowNode and not the CpsFlowExecution, should we try to call
+            // CpsVmExecutorService.reportProblem or CpsFlowExecution.croak to kill the build right away?
+            LOGGER.log(Level.WARNING, "Unable to load FlowNode or CpsFlowExecution when completing " + this + ", which is likely to cause its execution to hang indefinitely", x);
```

Comment:
We do not have any tests in this plugin which enter this `catch` block, and the only case where I ever saw it was while manually creating broken Pipelines while investigating https://github.com/jenkinsci/workflow-api-plugin/pull/349. I think for now it is good enough to just update the log message, and if we see it come up when investigating stuck Pipelines in the future, then it might be worth investigating further.